tests/test_data/test_blending.py

```diff
@@ -0,0 +1,41 @@
+import torch
+
+from mmaction.datasets.blending_utils import CutmixBlending, MixupBlending
```

Comment:
`from mmaction.datasets import CutmixBlending, MixupBlending` when `CutmixBlending, MixupBlending` are added in `__all__`